### PR TITLE
17-monitor-symptoms.md

### PR DESCRIPTION
**Pull request by siwei-wang**

### CODE REVIEW

**Comment by jaysonvirissimo** [comment]
You have to prepend the file name with a number followed by a hyphen and then the name or else you'll get the following error at build:
```
Liquid Exception: monitorsymptoms.md doesn't match the "99-some-file-name.md" name format (in _sections/act_and_prepare/en/02-do/monitorsymptoms.md) in act-and-prepare-en.md
```

File: _sections/act_and_prepare/en/02-do/monitorsymptoms.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Japan: www.coronatracker.com   
+
+
+
+
+
+
+
+
+
```
Comment (jaysonvirissimo) [inline _sections/act_and_prepare/en/02-do/monitorsymptoms.md]: We don't need quite so many empty newlines here. One should suffice.

Comment (siwei-wang) [reply _sections/act_and_prepare/en/02-do/monitorsymptoms.md]: just removed all empty lines

**Comment by jaysonvirissimo** [comment]
We've gotta change the actual name of the file, not just the PR title. I don't have repo privileges to do it myself, but whoever is going to merge this just need to add a commit after running `mv monitor-symptoms.md 17-monitor-symptoms.md` before merging.

File: _sections/act_and_prepare/en/02-do/monitorsymptoms.md (hunk 1 above)
Comment (jaysonvirissimo) [reply _sections/act_and_prepare/en/02-do/monitorsymptoms.md]: LGTM

**Comment by jaysonvirissimo** [comment]
That fixed the build @siwei-wang! The whole thing LGTM now.

**Comment by nditada** [comment]
@siwei-wang thanks a ton for this contribution, it's great. There are few issues with markdown and the missing tags for the table of content and the title. Can you please use the link avobe with the preview to verify what needs fixing? Also, for the titles and TOC, please compare with another file

**Comment by siwei-wang** [comment]
Ok, it looks passable to my eyes. Please help me check if there is anything else I should fix? Thank you all!

**Comment by jaysonvirissimo** [comment]
"Shortness of breath" still renders as **bolded** @siwei-wang:
Screen Shot 2020-03-27 at 3 23 53 PM

**Comment by siwei-wang** [comment]
fixed it

**Comment by siwei-wang** [comment]
Any more comments/suggestions before we deploy?

**Comment by jmcmurry** [comment]
This looks good to me (pending all ok on technical front). That said, this is a piece of data that is going to be pretty volatile moving forward as knowledge emerges. I suggest we simply link out to the authoritative sources as it is too difficult to be on top of it all the time.

**Comment by siwei-wang** [comment]
I agree it is a rapidly changing situation, but before another WHO report comes out, this is the best statistics available (at the scale of tens of thousands cases). We can amend it later if there are any significant updates.

**Comment by nditada** [comment]
@siwei-wang Let's do this: we'll keep this while you can take care of updating it with authoritative sources, if it gets outdated and you are busy, we'll replace the content for a link.

**Pull request merged**